events/Dockerfile

```
@@ -4,7 +4,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD events /src/events
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/events && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet . 
```

Please remove the trailing whitespace

```suggestion
RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```